apps/indexer/config/config.exs

```diff
@@ -14,7 +14,7 @@ block_transformers = %{
 configured_transformer = System.get_env("BLOCK_TRANSFORMER") || "celo"
 
 port =
-  case System.get_env("PORT") && Integer.parse(System.get_env("PORT")) do
+  case System.get_env("HEALTH_CHECK_PORT") && Integer.parse(System.get_env("HEALTH_CHECK_PORT")) do
```

Comment:
Can we keep the old env var? I wanted to reuse the existing env var as it's not used in the indexer.

Reply:
We can keep it for now if changing it will make deploying harder, but I think it should not be "PORT" because the "default" mode for running blockscout is running both the indexer and web server at the same time.